tools/build_script_generator/scons/resources/build_target.py.in

```diff
@@ -37,7 +37,8 @@ def build_target(env, sources):
 	env.Alias("size", env.Size(program))
 	%% if core.startswith("cortex-m")
 	env.Alias("itm", env.OpenOcdItm())
-	env.Alias("gdb", env.OpenOcdGdb(program))
```

Comment:
If you remove the current `gdb` is there an option for auto completing these commands

( :confused: or is it only me using bash ...)

Or is there any other way one can see in the command line available commands?

Reply:
`scons -h`, but we haven't implemented [the `Help()` function](https://scons.org/doc/HTML/scons-user/ch09.html), so there's nothing to show… I'll be reworking build system support in a while.